jujugui/static/gui/src/app/components/inspector-change-version/test-inspector-change-version.js

```diff
@@ -0,0 +1,202 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+var juju = {components: {}};
+var testUtils = React.addons.TestUtils;
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+describe('InspectorChangeVersion', function() {
+
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('inspector-change-version', function() { done(); });
+  });
+
+  it('can display an empty versions list', function() {
+    var changeState = sinon.stub();
+    var service = sinon.stub();
+    var setCharm = sinon.stub();
+    var getCharm = sinon.stub();
+    var getAvailableVersions = sinon.stub().callsArgWith(1, ['cs:django']);
+    var shallowRenderer = jsTestUtils.shallowRender(
+        <juju.components.InspectorChangeVersion
+          changeState={changeState}
+          charmId="cs:django"
+          service={service}
+          setCharm={setCharm}
+          getCharm={getCharm}
+          getAvailableVersions={getAvailableVersions} />, true);
+    shallowRenderer.getMountedInstance().componentDidMount();
+    var output = shallowRenderer.getRenderOutput();
+    assert.deepEqual(output,
+      <div className="inspector-change-version">
+        <div className="inspector-change-version__current">
+          Current version:
+          <div className="inspector-change-version__current-version"
+            role="button" tabIndex="0"
+            onClick={output.props.children[0].props.children[1].props.onClick}>
```

Comment:
Yep, I guess it's less of an assertion, but rather allowing everything else to be checked.